src/bundle/Resources/views/themes/admin/content/modal/draft_conflict.html.twig

```diff
@@ -21,21 +21,18 @@
             </div>
             <div class="modal-body">
                 <div class="ez-modal-body__main">
-                    <p class="ez-modal-body__main-content">
                         {% set number = conflicted_drafts|length %}
```

Comment:
I have an impression that after removing this tag you have bad indents in next lines